osidb/migrations/0060_remove_mitigated_by.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.14 on 2022-09-27 08:44
+
+from django.db import migrations, models
+import osidb.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osidb', '0059_flaw__alerts'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='flaw',
+            name='mitigated_by',
+        ),
+        migrations.RemoveField(
+            model_name='flawhistory',
+            name='mitigated_by',
+        ),
+    ]
```

Comment:
This is not N-1 safe, see https://develop.sentry.dev/database-migrations/#deleting-columnstables for safe ways to remove fields